libs/filabridge/include/private/filament/UibStructs.h

```diff
@@ -288,8 +288,11 @@ struct PerRenderableBoneUib { // NOLINT(cppcoreguidelines-pro-type-member-init)
     };
     BoneData bones[CONFIG_MAX_BONE_COUNT];
 };
-static_assert(sizeof(PerRenderableBoneUib) <= 16384,
-        "PerRenderableUibBone exceed max UBO size");
+
+// ES3.0 only guarantees a max UBO size of 16 KiB, but users may wish to exceed the minspec
+// by adjusting CONFIG_MAX_BONE_COUNT.
+static_assert(sizeof(PerRenderableBoneUib) == CONFIG_MAX_BONE_COUNT * 64,
```

Comment:
Maybe replace 64 with `sizeof(BoneData)`? That makes me question if we really need this assertion now

Similar suggestion for the morph targets.